tests/functional/object/mcg/test_noobaa_endpoint_keda.py

```
@@ -0,0 +1,235 @@
+        # 3. Wait for endpoints to stabilize at minCount
+        logger.info(f"Waiting for endpoints to stabilize at {self.MIN_ENDPOINT_COUNT}")
+        try:
+            for endpoint_pods in TimeoutSampler(
+                timeout=300,
+                sleep=30,
+                func=get_pods_having_label,
+                label=constants.NOOBAA_ENDPOINT_POD_LABEL,
+                namespace=namespace,
+            ):
+                count = len(endpoint_pods)
+                logger.info(f"Endpoint pod count: {count}")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)